src/viewer.ts

```diff
@@ -165,6 +197,8 @@ export class Viewer {
 
   onVimLoaded (vim) {
     for (let i = 0; i < vim.meshes.length; ++i) {
+      vim.meshes[i].castShadow = true;
```

Comment:
I think this was omitted previously due to performance reasons, but I don't recall. Let's make it an option in a follow-up PR. Which suggests that the logic has to be a bit different.